comp/metadata/inventoryagent/README.md

```diff
@@ -62,6 +62,9 @@ The payload is a JSON dict with the following fields
     `network_config.enabled` config option in `system-probe.yaml`).
   - `feature_oom_kill_enabled` - **bool**: True if the OOM Kill check is enabled for System Probe (see: `system_probe_config.enable_oom_kill` config option in `system-probe.yaml`).
   - `feature_tcp_queue_length_enabled` - **bool**: True if TCP Queue Length check is enabled in System Probe (see: `system_probe_config.enable_tcp_queue_length` config option in `system-probe.yaml`).
+  - `python` - **object**:
+    - `python_version` - **string**: The version of the embedded python.
+    - `python_packages` - **dict of string to string**: All the python packages installed in the embedded python with their versions.
```

Comment:
How do you plan to store this in REDAPL ? We can't have a row per  python packages since we want a single row per Agent. Are we storing a JSON blob and using SQL JSON operator ?

If not, maybe a dedicated table would be more valuable, WDYT ?

I guess the question is: how do you plan to query this ?

Reply:
That would most likely be a dedicated REDAPL table yes, used by fleet services to know what is installed/what is expected to be installed

Reply:
We match each payload to a single table. If this is aimed at another table we should create a dedicated payload for this.